MapboxNavigation/RouteViewController.swift

```diff
@@ -9,7 +9,10 @@ public class NavigationPulleyViewController: PulleyViewController {}
 
 @objc(MBRouteViewControllerDelegate)
 public protocol RouteViewControllerDelegate {
-    func routeViewControllerDidCancelNavigation(_:RouteViewController)
+    @objc optional func routeViewControllerDidCancelNavigation(_:RouteViewController)
+    func navigationMapView(_ mapView: NavigationMapView, routeStyleLayerWithIdenitier identifier: String, source: MGLSource) -> MGLStyleLayer?
+    func navigationMapView(_ mapView: NavigationMapView, routeCasingStyleLayerWithIdenitier identifier: String, source: MGLSource) -> MGLStyleLayer?
+    func navigationMapView(_ mapView: NavigationMapView, shapeDescribing route: Route) -> MGLShape?
```

Comment:
I know this protocol has always been undocumented, but let’s start by adding documentation for any new methods.